src/backend/distributed/commands/utility_hook.c

```diff
@@ -165,6 +177,38 @@ multi_ProcessUtility(PlannedStmt *pstmt,
 		return;
 	}
 
+	PG_TRY();
+	{
+		utilityHookLevel++;
+
+		bool executedDDLJob = citus_ProcessUtility(pstmt, queryString, context,
```

Comment:
we already have another function named `CitusProcessUtility`. We should come up with a better name for either of these

Reply:
yeah thinking about the same thing
https://github.com/citusdata/citus/pull/4538#discussion_r560016567